spec/system/task_spec.rb

```diff
@@ -1,27 +1,33 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { create(:project) }
+  let(:task) { create(:task, project: project) }
```

Comment:
Factoryの指摘事項を対応することで省略可能です。
```suggestion
  let(:task) { create(:task) }
```